### PR TITLE
added tab to get only user's wikis

### PR DESCRIPTION
**Pull request by AbdullahAlger**
Still have to add collaborators, but added a useful tab first.

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class UsersController < ApplicationController
+  def index
+    @wikis = current_user.wikis.all
```
Comment (eliotsykes) [inline app/controllers/users_controller.rb]: The `.all` can be safely dropped, `current_user.wikis` is equivalent.

File: config/routes.rb [hunk 2]
```diff
@@ -16,6 +15,10 @@
 
   resources :wikis
 
+  resources :users, only: [:index] do
+    resources :wikis
```
Comment (eliotsykes) [inline config/routes.rb]: Not sure wikis need to be nested under users. Wikis as a standalone resource on line 16 seem fine.

Comment (eliotsykes) [inline config/routes.rb]: Ok forget comment above, I see the purpose now, the app has a dedicated page for showing a user's wikis.

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (AbdullahAlger) [inline app/controllers/users_controller.rb]: Done.

**Pull request merged**